### PR TITLE
translations: downloaded lilac translations

### PR DESCRIPTION
**Pull request by black-redoc**
**description:**
Updated translations and downloaded from transifex.

**changes:**
updated i18n and static files from:
- lms/static
- cms/static
updated locales from:
- conf/locale

**how to test:**
check the pages and change the translations with the language selector into header.

**jira:**
https://edunext.atlassian.net/browse/PS2021-1374

### CODE REVIEW

File: .tx/config [hunk 1]
```diff
@@ -78,3 +78,4 @@ file_filter = conf/locale/<lang>/LC_MESSAGES/djangojs.po
 source_file = conf/locale/en/LC_MESSAGES/djangojs.po
 source_lang = en
 type = PO
+
```
Comment (mariajgrimaldi) [inline .tx/config:81]: is this necessary?

Comment (MoisesGSalas) [reply .tx/config:81]: No, this file should stay the same.

**Review by MoisesGSalas — CHANGES_REQUESTED** [review]
There is a lot of files missing. Filtering by `*.po` extension only throws 2 files: `conf/locale/th/LC_MESSAGES/django.po` and `conf/locale/th/LC_MESSAGES/djangojs.po`

**Comment by black-redoc** [comment]
> There is a lot of files missing. Filtering by `*.po` extension only throws 2 files: `conf/locale/th/LC_MESSAGES/django.po` and `conf/locale/th/LC_MESSAGES/djangojs.po`

What do you mean?

**Review by mariajgrimaldi — APPROVED** [review]

**Pull request merged**